.github/workflows/main.yml

```diff
@@ -43,7 +43,7 @@ jobs:
 
     - name: 🔨 Build & Test
       run: |
-        stack --no-terminal test --no-run-tests --bench --haddock --no-haddock-deps --no-run-benchmarks --flag cardano-addresses:release --flag cardano-addresses-cli:release
```

Comment:
I believe the rationale for adding `--no-run-tests` was this https://github.com/input-output-hk/cardano-addresses/pull/90/files#r522033013

_(but perhaps it's a bit confusing from the first glance)_

Reply:
It's very confusing.

Reply:
I looked at the master branch CI build logs and it turns out the tests do actually run.
They fail, but the result is still green.
At least with this change, if tests fail then CI turns red.